### DISCUSSION TITLE
How do analog USB modules compare?

### DISCUSSION THREAD

**Discussion by stevedee** [Nachtzuster/BirdNET-Pi#255] [General]
I've been giving some thought to this question and have just tested 4 popular types.
There are lots of low price analog USB modules available for use with typical electret microphone capsules, but none (as far as I know) are designed for use with BirdNet-Pi.

They are typically aimed at users that need a head-set interface for a laptop or pc. The mic inputs are designed to work with a wide range of electret microphones in situations where the mic is close to the users mouth.

For BirdNet-Pi we are looking for a very sensitive microphone channel, where our bird of interest is maybe 10's of metres away. Frequency response may only need to cover a more limited range, maybe 100 or 200Hz up to (say) 12kHz.

I carried out some simple sensitivity tests on 4 models of audio USB modules and found that one of the cheapest was actually the most sensitive.

The main take-away from this is that we should be able to get higher sensitivity from our systems by tuning the analog USB adapters to match our selected electret microphone, rather than by spending more money on what's thought to be a better module.

Details here: http://captainbodgit.blogspot.com/2024/12/birdnet-pi-usb-audio-module-testing.html

BTW: I also noticed that the system spectrogram displays less noise when using the slower USB sockets when compared the fast (blue) ones.

**Comment by alexbelgium** [comment]
Very nice website and content
